packages/arb-token-bridge-ui/src/components/MainContent/MainContent.tsx

```diff
@@ -99,16 +98,6 @@ export function MainContent() {
             <TransferPanel />
           </motion.div>
         </AnimatePresence>
-
```

Comment:
instead of removing the code, would commenting it out be better if we foresee its return?